exporter/api.py

```diff
@@ -37,8 +38,30 @@ def sanitize(title):
                 id=learning_resource.id,
                 title=sanitize(learning_resource.title),
             )
-            with open(os.path.join(tempdir, filename), 'w') as f:
+            type_name = learning_resource.learning_resource_type.name
+            try:
+                os.mkdir(os.path.join(tempdir, type_name))
+            except OSError:
+                pass
```

Comment:
how can you be sure that the directory exists if you just pass here?
if you do not have rights to create the directory, it will still give you `OSError`:

```
In [1]: import os

In [2]: os.mkdir('/usr/bin/foogiovanni')
---------------------------------------------------------------------------
OSError                                   Traceback (most recent call last)
<ipython-input-2-bdef8fc69048> in <module>()
----> 1 os.mkdir('/usr/bin/foogiovanni')

OSError: [Errno 13] Permission denied: '/usr/bin/foogiovanni'
```